doc/manual/manual.tex

```diff
@@ -5983,19 +5979,25 @@ \subsubsection{Tracking finite strain}
 
 \begin{figure}
     \centering
-    \includegraphics[width=0.5\textwidth]{cookbooks/finite_strain/finite_strain.pdf}
+    \includegraphics[width=0.75\textwidth]{cookbooks/finite_strain/finite_strain.pdf}
     \caption{Accumulated finite strain in an example convection model, as described in Section 
-             \ref{sec:finite-strain}.}
+             \ref{sec:finite-strain} at a time of 67.6~Ma. Top panel: Temperature distribution. Bottom panel:
+             Natural strain distribution. Additional black crosses are the scaled eigenvectors of the
+             stretching tensor $\mathbf L$, showing the direction of stretching and compression.}
```

Comment:
So the rotation matrix is effectively unimportant to find the major stress axes?